### ISSUE TITLE
Import Config and Checkbox state does not work, throws error

### ISSUE THREAD

**Issue by KataFF** [openscopeproject/InteractiveHtmlBom#333]
I am assembling a board complicated enough that it takes multiple days to get it done. Therefore I'm making heavy use of the save/restore checkbox state feature, to keep track of what elements need yet to be placed.

On a DELL Inspiron P94G001 2-in-1 Chromebook, with the stock Chrome OS, in Chrome version 106.0.5249.112 (Official Build) (64-bit) importing checkbox state does not work through "Disk transfer icon" > "Config and checkbox state" > "Import"; The checkboxes do not change.
The file selection dialog opens just fine, however shows no other files than the "*.settings.json" file, and folders. 

I opened up the (JavaScript) Developer Console with Ctrl+Shift+J, and here I can see the following error getting thrown the exact moment I try to import the file:
```
ibom.html:1335 Uncaught ReferenceError: showFootprints is not defined
    at overwriteSettings (ibom.html:1335:3)
    at FileReader.<anonymous> (ibom.html:1294:7)
overwriteSettings @ ibom.html:1335
(anonymous) @ ibom.html:1294
load (async)
(anonymous) @ ibom.html:1262
```

Note that the checkboxes do seem to be saved automatically (localStorage or something?), however upon closing tab and reloading they seem to be at some old state, losing progress I checked before closing.

**Comment by KataFF** [comment]
Also note that the ibom.html file was generated using newest KiCad 6.0.8 on up-to-date Manjaro Linux, and later transferred(copied) to the Chromebook by the means of a USB Pendrive.

**Comment by qu1ck** [comment]
I don't know what showFootprints is, that name doesn't exist in plugin code. You either have an old version of the plugin or some custom code in user.js or a different fork. Can't help you there.

**Comment by qu1ck** [comment]
Nvm, I was looking at something else. Will fix soon.

**Comment by KataFF** [comment]
Gorgeous, your nickname really fits in this case :)
Btw thanks for the involvement
Now figuring out how to install the git version of the plugin into KiCad and will report if it works.

**Comment by qu1ck** [comment]
I don't plan a new release soon, you may have to wait a while. If you look at the commit, it's just removal of the offending line.
So you can 
a) remove plugin from plugin manager and follow manual installation instructions on [wiki](https://github.com/openscopeproject/InteractiveHtmlBom/wiki/Installation)
b) edit the plugin installed by plugin manager and remove the line yourself

Plugin manager has separate path for plugins, it's under KICAD6_3RD_PARTY environment variable. You can find where that points by going to `configure paths` in kicad preferences menu.

**Comment by KataFF** [comment]
Thanks for the tips. I went for a modified b) approach - without re-generating the iBOM, I commented out `showFootprints(...);` function call from the already generated ibom.html on the Chromebook.

Now importing config & checkbox state works as intended on the Chromebook (the exact setup described in OP), and no errors are thrown to the Developer Console.

**Comment by KataFF** [comment]
As for the plugin itself on Manjaro KiCad install, I did the full b) solution.
Here's what I did exactly, for reference & so that others with the same problem can apply a bugfix:
1. I went to the KICAD6_3RD_PARTY path by copying it and pasting in the File Manager address bar, and entered the InteractiveHtmlBom plugin directory.
A bash one liner for this would be: `cd ~/.local/share/kicad/6.0/3rdparty/plugins/org_openscopeproject_InteractiveHtmlBom/` 
2. Opened up terminal there, and searched for the leftover function call:
```
$ grep -rn showFootprints
web/util.js:439:  showFootprints(settings.show_footprints); 
```
3. Opened up the mentioned file in a text editor (VSCodium, nano, whatever you like) and deleted line 439.

Again, thank you @qu1ck very much for this amazing plugin and for the quick bugfix.
